i18n_scripts/split_translation_files.py

```diff
@@ -0,0 +1,49 @@
+import os, localizable
+from collections import OrderedDict
+
+
+def separate_modules_translations(modules_dir, languages_dirs):
+    translations = {}
+    for lang_dir in languages_dirs:
+        file_path = os.path.join(modules_dir, 'I18N', 'I18N', lang_dir, 'Localizable.strings')
+        lang_list = localizable.parse_strings(filename=file_path)
+        for line in lang_list:
+            module_name, _, key_remainder = line['key'].partition('.')
+            module_name = module_name
+            new_line = {
+                'key': key_remainder,
+                'value': _escape(line['value']),
+                'comment': line['comment']
+            }
+            translations.setdefault(module_name, {}).setdefault(lang_dir, []).append(new_line)
```

Comment:
This line is rather complex, with the `for lang in langs` loop it should help to simplify it.

Reply:
This line is rather complex, with the `for lang in langs` loop it should help to simplify it:

```suggestion
            translations.setdefault(module_name, [])
            translations[module_name].append(split_entry)
```

Reply:
Semplified